ckanext/harvest/harvesters/ckanharvester.py

```diff
@@ -36,6 +35,11 @@ def _get_search_api_offset(self):
     def _get_content(self, url):
 
         headers = {}
+
+        user_agent = self.config.get('user_agent')
+        if user_agent:
+            headers['User-Agent'] = user_agent
```

Comment:
whats the default user agent if not set?

Reply:
We don't set it. Probably `python-requests/<version>`